<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelVector">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>511</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>140</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Текущий вектор : </string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>332</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="resizePB">
       <property name="text">
        <string>resize</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="sizePB">
       <property name="text">
        <string>size</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>481</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="atPB">
         <property name="text">
          <string>at</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="capacityPB">
         <property name="text">
          <string>capacity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearPB">
         <property name="text">
          <string>clear</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dataPB">
         <property name="text">
          <string>data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="emptyPB">
         <property name="text">
          <string>empty</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="erasePB">
         <property name="text">
          <string>erase</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="maxSizePB">
         <property name="text">
          <string>maxSize</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="popBackPB">
         <property name="text">
          <string>popBack</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushBackPB">
         <property name="text">
          <string>pushBack</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reservePB">
         <property name="text">
          <string>reserve</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
